manifests/overlays/prod/secrets/clusters/paas.stage.psi.redhat.com.enc.yaml

```diff
@@ -10,7 +10,7 @@ type: Opaque
 stringData:
     name: paas.stage.psi.redhat.com
     config: ENC[AES256_GCM,data:VCLAzJI24C78EEBMfN/bQ2+HKxTxXyDjWS4lp6VuYC1KVSRW/7fVeHojUfl4o/HypAzRnBByZ9lO98rgsTZxDAbHzpnSxstkuERhV7CgUbJ1jySIYCpS5YR401guwqyVKiwt88rXTRmP07Ddl+qfcNym1pkmw05q7bYh5J1jiI7rkTcNZzod1DGznNryk/nTHPa5RXtinpJgQjnUyi1xmY/6PoT3Vuc8IoT2GC3FY6sXonGHkKjljs8Oh2R4zUfWzWeIYAE06sb0ZrTdLc06Ojqa/bh+GPM28roJa/96QRS16KS01YuZXYmgAgQwdOKTuyqHRw5TXVdStetHcwew/wJonf9OKfP5n4LF5IH/cplVkg/u/U0i+3GcSG4D9pHUicMNSf2EAPsrPgZcf3Xx/XVewV//tgfFt/+DB0YhzYlBXQG6F2Kpr1gzv6Sa340NfrBhrv8wljYylTwdpYQSJb72py9q+VUjh3Pu18XBaIfuwwVBvka1CEK25b8ulYdSCF5+YYMAiKZZiVyVYHdkaIkttBOxiNmPwmRechByxVt52R0y+CLfNE3SEVFvBdwj1/xvEhtQqGRXN+iA9rUOszmK5xwg3XRL2lVLCwsnH2x6xeNSwT+sri53f9uwHla3QfypCbPP1YgYliR5e6dr6wS4ftk4eoIVvnRdsOxXZVA3QjNjRLuBEmHe3aGPC8gMs+1Y6VHXtttIx+qgeJH/ppvsKUBCPY22OVNDiC2bWCH1iOAwtAUV3S3keP2cfmvLpnzFn4se8l3vqLGatk8ofC2gFa7KzitoSx4mTBNy3vw8D5SWGkJW62gdlZMyM4yyyhwa56xEzhgpXvA0XJUjR105TCZa9U7ZU7Jh5Dhq+u6BHJzMb0naayZSOewlb84lGeSZpRSOI3e9y7aKC2IkOogVj4GEFbcxMW6Z7HfgYVmxSmw2QF1aLrbhpWkO63hP73qQXYkzYKhln65RFKgPuavAPRe9Q0wyo7+GkMGx1KJ+sVFvPHUBH1nMjchwKQmXdVEhGkcSqDtXIK0B9Ziol7YhFZnYWxEPPOWDk7WcItVZGA0hpZSiVdit0hj4C/wCg1mSolZCeqjCmkgML3BnZrJ5IqvxFlB3IxTe7SXFvYRZopXJALgDlhgaaeVLx5z9kMAZNiqYI6yQw3x/p7WJvJWjVlenY6u21QzU1w1Y5KdiVYEQhYw/SNsR8z7aHESTIx6PDSEDbqWsE0KUb+0Fjlfx3NQaTFBQPnRrkI6sWZ1ixlQ=,iv:4tnHCEUasLXqvkxUM4PgzlCLirZwf+yHKWSz6At15hE=,tag:RwKW3AU1EzouW1KZ+UWrCg==,type:str]
-    namespaces: dh-stage-jupyterhub,dh-stage-argo,dh-stage-message-bus
+    namespaces: dh-stage-jupyterhub,dh-stage-argo,dh-stage-message-bus,dh-stage-ingest
```

Comment:
you have to edit this field via sops (or decrypt it, edit it, then re-encrypt) 

do: 
```
$ sops paas.stage.psi.redhat.com.enc.yaml 
```

This will open an editor, make changes, save and exit

Reply:
To salvage things right now, you may want to do something like `sops --ignore-mac paas.stage.psi.redhat.com.yaml`

Reply:
I wondered if that might be the case. Fixed.